business_requirement_from_support/wizard/br_from_issue_view.xml

```diff
@@ -0,0 +1,41 @@
+<openerp>
+   <data>
+       <!-- Make BR from issue view -->
+       <record id="view_create_br_from_issue" model="ir.ui.view">
+          <field name="name">create.br.form</field>
+          <field name="model">br.issue</field>
+          <field name="arch" type="xml">
+            <form string="Business Requirement from Issue" >
```

Comment:
Tag closing is not consistent (sometimes there's a space before `>` or `/>`, sometimes not)